config.yml.dist

```diff
@@ -98,7 +98,16 @@ media:
     fr: Droits d'auteur gérés par %s
     de: Urheberrecht gehandhabt von %s
     en: Copyright handled by %s
-
+  media_url_polyfill:
+    udb2:
+      enabled: true
+      legacy_url: https://media.uitdatabank.be/
+      new_url: https://images-dev.uitdatabank.be/
```

Comment:
You could just make it `url`, because of `legacy_url` it's clear that this is the new one.